pkg/operator/controller/garden/components.go

```diff
@@ -393,3 +395,30 @@ func fetchKubeconfigFromSecret(ctx context.Context, c client.Client, key client.
 
 	return kubeconfig, nil
 }
+
+func (r *Reconciler) newIstio(garden *operatorv1alpha1.Garden) (istio.Interface, error) {
+	var annotations map[string]string
+	if settings := garden.Spec.RuntimeCluster.Settings; settings != nil && settings.LoadBalancerServices != nil {
+		annotations = settings.LoadBalancerServices.Annotations
+	}
+
+	return sharedcomponent.NewIstio(
+		r.RuntimeClientSet.Client(),
+		r.ImageVector,
+		r.RuntimeClientSet.ChartRenderer(),
+		namePrefix,
+		v1beta1constants.DefaultSNIIngressNamespace,
+		v1beta1constants.PriorityClassNameGardenSystem500,
```

Comment:
Please adapt https://github.com/gardener/gardener/blob/master/docs/development/priority-classes.md#priorityclasses-for-garden-control-plane-components